src/Language/Haskell/Liquid/CmdLine.hs

```diff
@@ -242,12 +256,41 @@ parsePragma   :: Located String -> IO Config
 parsePragma s = withArgs [val s] $ cmdArgsRun config
 
 ---------------------------------------------------------------------------------------
+withCabal :: Config -> IO Config
+---------------------------------------------------------------------------------------
+withCabal cfg
+  | cabalDir cfg = withCabal' cfg
+  | otherwise    = return cfg
+
+withCabal' cfg = do
+  putStrLn $ "addCabalDirs: " ++ tgt
```

Comment:
We should try to respect the verbosity settings, using `whenNormal` (or perhaps in this case `whenVerbose`) to guard `print` expressions.